### ISSUE TITLE
Host configuration don't work at instalation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [paleg/eventlog-to-syslog#86]
```

Installation is done in dos prompt with elevated privileges :

evtsys.exe -i -h 192.168.220.1

Process reports no errors, but the key LogHost is empty. (if populated by hand 
it works).

v4.5.0x64, OS 2008R2

Any advices are welcome.

Tanks
```

Original issue reported on code.google.com by `yipik...@gmail.com` on 2 Oct 2013 at 4:16

**Comment by GoogleCodeExporter** [comment]
```
Thanks for letting me know. I missed it in my last change. It has been 
corrected in the latest build (v4.5.1).
```

Original comment by `sherwin....@gmail.com` on 3 Oct 2013 at 5:10
- Changed state: **Fixed**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for your reactivity !

Cédric

```

Original comment by `yipik...@gmail.com` on 3 Oct 2013 at 3:41
